features/302.ngsild.BigDataFlink.feature

```diff
@@ -0,0 +1,72 @@
+Feature: test tutorial 302 - Big Data Analysis (Flink)
+
+  url: https://ngsi-ld-tutorials.readthedocs.io/en/latest/big-data-flink.html
+
+  git-clone: https://github.com/FIWARE/tutorials.Big-Data-Flink.git
+  git-directory: /tmp/tutorials.Big-Data-Flink
+  shell-commands: git checkout NGSI-LD ; ./services create  ; ./services start
+  clean-shell-commands: ./services stop
+
+Background:
+    Given  I set the tutorial 302 LD - Big Data Flink
+
+    # COMPILING A JAR FILE FOR Flink --
+    Scenario: Compiling a Jar File For Flink
+       Given I prepare the script create-flink-jar.sh to be run
+       And   I set the environ variable WORKING_DIR to "cosmos-examples" under git
```

Comment:
environ -> environment

Reply:
"environ" is accepted from feature "201. Introduction to IoT Sensors" some months ago. I suggest, after merging this branch creating a new PR in order to change the label "I set the environ variable...." and the functions in order to break nothing.

Reply:
NTC